kie-archetypes/kie-kjar-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>\${groupId}</groupId>
+  <artifactId>\${artifactId}</artifactId>
+  <version>\${version}</version>
+  <packaging>kjar</packaging>
+  <name>\${artifactId}</name>
+
+  <properties>
+    <version.org.kie>${project.version}</version.org.kie>
+    <version.org.junit>4.12</version.org.junit>
+    <version.com.thoughtworks.xstream>1.4.10</version.com.thoughtworks.xstream>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
```

Comment:
could you please remove the dependencies? it should not be there for empty projects and only added when needed.